src/main/java/ledger/database/storage/ISQLiteDatabaseTransaction.java

```diff
@@ -0,0 +1,445 @@
+package ledger.database.storage;
+
+import ledger.database.IDatabase;
+import ledger.database.enity.*;
+import ledger.exception.StorageException;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by CJ on 10/11/2016.
+ */
```

Comment:
By no means is this something we need to worry about now but I think we can eventually break a few of these methods up and separate the chunks of logic.